src/courtroom.cpp

```diff
@@ -2107,6 +2113,23 @@ void Courtroom::on_cycle_clicked()
 void Courtroom::cycle_shout(int p_index)
 {
   int n = ui_shouts.size();
+
+  // Check if any shout button is available to begin with
+  int i;
+  for (i = 0; i < n; i++)
+  {
+    if (shouts_enabled[i])
+      break;
+  }
+  if (i == n)
+  {
+    call_notice(
+        "Fatal error: no shout button images found in current theme folder.");
```

Comment:
...Is this that fatal of an error?

Reply:
It causes an infinite loop, although not sure if crashing out is the best way to solve.